app/src/main/java/pl/gittobefit/WorkoutDisplay/fragments/ListOfTrainings.java

```diff
@@ -22,34 +25,66 @@
 
 public class ListOfTrainings extends Fragment
 {
-    public ListOfTrainings() {
-    }
+    private RecyclerView trainingList;
+    private LinearLayout loading;
+    TrainingListAdapter trainingListAdapter;
+    InitiationTrainingDisplayLayoutViewModel model;
 
+    public ListOfTrainings() { }
 
     @Override
-    public void onCreate(@Nullable Bundle savedInstanceState) {
+    public void onCreate(@Nullable Bundle savedInstanceState)
+    {
         super.onCreate(savedInstanceState);
     }
 
     @Nullable
     @Override
-    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState)
+    {
         View view = inflater.inflate(R.layout.fragment_list_of_trainings, container, false);
         return view;
     }
 
     @Override
-    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState)
+    {
         super.onViewCreated(view, savedInstanceState);
-        InitiationTrainingDisplayLayoutViewModel model = new ViewModelProvider(requireActivity()).get(InitiationTrainingDisplayLayoutViewModel.class);
+        trainingList = getView().findViewById(R.id.list_of_trainings);
+        loading = getView().findViewById(R.id.list_traing_loading);
+        model = new ViewModelProvider(requireActivity()).get(InitiationTrainingDisplayLayoutViewModel.class);
+        trainingList.addItemDecoration(new DividerItemDecoration(getContext(), DividerItemDecoration.VERTICAL));
+        trainingListAdapter = new TrainingListAdapter(model.getTrainingWithForms(), this);
+        trainingList.setAdapter(trainingListAdapter);
+        trainingList.setLayoutManager(new LinearLayoutManager(getContext(), LinearLayoutManager.VERTICAL, false));
+        User.getInstance().getSynchroniseTraining$().observe(getViewLifecycleOwner(), new Observer<User.SynchroniseTraining>()
+        {
+            @Override
+            public void onChanged(User.SynchroniseTraining synchroniseTraining)
+            {
+                if(User.SynchroniseTraining.No_Synchronise == synchroniseTraining || synchroniseTraining == User.SynchroniseTraining.Synchronise_Success)
```

Comment:
Generalnie, nie przyczepiałem się do tego, bo sam trochę to olałem, ale jak robimy enumy to poszczególne wartości mogłyby być nazywane tak samo jak stałe, czyli dużymi literami